src/finchlite/autoschedule/einsum.py

```diff
@@ -0,0 +1,262 @@
+from collections.abc import Callable
+from typing import Any
+
+import finchlite.finch_einsum as ein
+from finchlite.algebra import init_value, is_commutative, overwrite
+from finchlite.finch_logic import (
+    Aggregate,
+    Alias,
+    Field,
+    Literal,
+    LogicExpression,
+    LogicNode,
+    MapJoin,
+    Plan,
+    Produces,
+    Query,
+    Reformat,
+    Relabel,
+    Reorder,
+    Table,
+)
+from finchlite.symbolic import gensym
+
+
+class EinsumLowerer:
+    def __call__(self, prgm: Plan) -> tuple[ein.Plan, dict[str, Any]]:
+        bindings: dict[str, Any] = {}
+        definitions: dict[str, ein.Einsum] = {}
+        return self.compile_plan(prgm, bindings, definitions), bindings
+
+    def reorder_einsum(
```

Comment:
do not use this function.

Reply:
`reorder_einsum` has been deleted. Hadn't noticed that it was still there.